examples/extensions/resolver.ts

```diff
@@ -0,0 +1,42 @@
+import { Resolver, Query, Mutation, Arg, Extensions } from "../../src";
+import { Logger } from "./logger.decorator";
+
+import { Recipe } from "./recipe.type";
+import { createRecipe, sampleRecipes } from "./recipe.helpers";
+
+@Resolver()
+export class ExampleResolver {
+  private recipesData: Recipe[] = sampleRecipes.slice();
+
+  @Extensions({ some: "data" })
+  @Query(returns => [Recipe])
+  async recipes(): Promise<Recipe[]> {
+    return await this.recipesData;
+  }
+
+  @Mutation()
+  addRecipe(
+    @Arg("title") title: string,
+    @Arg("description", { nullable: true }) description?: string,
+  ): Recipe {
+    const newRecipe = createRecipe({
+      title,
+      description,
+      ratings: [],
+    });
+    this.recipesData.push(newRecipe);
+    return newRecipe;
+  }
+
+  @Logger("This message will not be logged")
+  @Logger("It will be overridden by this one")
```

Comment:
We have the rules described in docs - I think it's enough to show a real use case of the feature to demonstrate when it's helpful

Reply:
Do you mean that this specific example of adding an extension twice on the same field should be remove and that showing it in docs is enough? If so, I have removed this example here: 531cc23820cf91f4c0ad26fcadb23afe9cf31ffd

Hope I got it properly